app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -0,0 +1,556 @@
+
+    async submitRegistration (registration) {
+      // TODO: isRegistered is not reactive because we're not using Vuex :( Improve this
+      this.doneRegistering = true
+
+      // TODO: Validate here too?
+      me.set('firstName', registration.firstName)
+      me.set('lastName', registration.lastName)
+      me.set('name', registration.name)
+      me.set('email', registration.email)
+      me.set('emails', registration.emails)
+      me.set('birthday', registration.birthday)
+      me.set('unsubscribedFromMarketingEmails', registration.unsubscribedFromMarketingEmails)
+      await me.save()
+    }
+  },
+
+  computed: {
+    ...mapGetters({
+      globalRankings: 'seasonalLeague/globalRankings',
+      clanRankings: 'seasonalLeague/clanRankings',
+      myClans: 'clans/myClans',
+      clanByIdOrSlug: 'clans/clanByIdOrSlug',
+      isLoading: 'clans/isLoading'
+    }),
+
+    currentSelectedClan () {
+      return this.clanByIdOrSlug(this.clanIdSelected) || null
+    },
+
+    selectedClanRankings () {
+      return this.clanRankings(this.clanIdSelected)
+    },
+
+    isRegistered () {
+      const emails = me.get('emails') || {}
+      const unsubscribed = me.get('unsubscribedFromMarketingEmails')
+      return (emails.generalNews || {}).enabled && !unsubscribed
+    },
+
+    canRegister: function () {
+      return !me.isAnonymous() && !this.isRegistered
+    },
+
+    // NOTE: `me` and the specific `window.me` are both unavailable in this template for some reason? Hacky...
+    firstName () { return me.get('firstName') },
+
+    lastName () { return me.get('lastName') },
+
+    name () { return me.get('name') },
+
+    email () { return me.get('email') },
+
+    emails () { return me.get('emails') },
+
+    birthday () { return me.get('birthday') },
+
+    unsubscribedFromMarketingEmails () { return me.get('unsubscribedFromMarketingEmails') }
+  }
+}
+</script>
+
+<template>
+  <main id="page-league-global">
+    <league-signup-modal
+        v-if="leagueSignupModalOpen"
+        @close="leagueSignupModalOpen = false"
+        @submit="submitRegistration"
+        :first-name="firstName"
+        :last-name="lastName"
+        :name="name"
+        :email="email"
+        :birthday="birthday"
+        :emails="emails"
+        :unsubscribed-from-marketing-emails="unsubscribedFromMarketingEmails"
+    >
+    </league-signup-modal>
+
+    <section class="row flex-row" style="margin-bottom: -100px;">
+      <img src="/images/pages/league/logo_codecombat_blitz.png" width="234" height="197" />
+    </section>
+
+    <section class="row flex-row" style="min-height: 600px;">
+      <div class="col-sm-5">
+        <h1 style="transform: rotate(-10deg);"><span class="esports-pink">COMPETITIVE </span><span class="esports-green">CODING </span><span class="esports-aqua">HAS </span><span class="esports-purple">NEVER </span><span class="esports-pink">BEEN </span><span class="esports-aqua">SO </span><span class="esports-green">EPIC</span></h1>
```

Comment:
Best practice to make all of the text in these UPPERCASE HEADERS actually Title Case and use `text-transform: uppercase` to render capitalized, because:
1. We could change our minds later about whether it should be upper case or title case more easily without having to mess around with locale strings.
2. Looks better in search engine result previews.

Reply:
Agree and excellent call out.